src/librustc_codegen_ssa/mir/mod.rs

```diff
@@ -88,6 +88,7 @@ pub struct FunctionCx<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> {
 }
 
 impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
+    #[inline(never)]
```

Comment:
why this?

Reply:
I was trying to get a complete backtrace and forgot to remove it.